lib/commands/push.ts

```diff
@@ -300,10 +348,28 @@ export default class PushCmd extends Command {
 					sdk,
 					opts,
 				};
-				await remote.startRemoteBuild(args);
+				const releaseId = await remote.startRemoteBuild(args);
+				if (releaseId) {
+					// Above we have checked that releaseTagKeys and releaseTagValues are of the same size
+					await Promise.all(
+						(_.zip(releaseTagKeys, releaseTagValues) as Array<
+							[string, string]
+						>).map(async ([key, value]) => {
+							await sdk.models.release.tags.set(releaseId, key, value);
+						}),
+					);
+				}
```

Comment:
I think we could add an `else` block here that prints a warning that the tags have not been set because the release ID could not be obtained. For example:
```
	if (releaseId) {
		...
	} else if (releaseTagKeys.length) {
		throw new Error(stripIndent`
			The remote build has started or concluded, but a release ID could not be parsed
			out of the builder's output. As a result, the release tags have not been set.`);
	}
```
Above I've used `Error` instead of `ExpectedError`. Not sure, but if the releaseId could not be parsed, this may actually be an _unexpected_ error worth reporting via Sentry.io.